CHANGELOG.md

```diff
@@ -137,7 +148,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [\#8628](https://github.com/cosmos/cosmos-sdk/issues/8628) Commands no longer print outputs using `stderr` by default
 * [\#9134](https://github.com/cosmos/cosmos-sdk/pull/9134) Renamed the CLI flag `--memo` to `--note`.
 * [\#9291](https://github.com/cosmos/cosmos-sdk/pull/9291) Migration scripts prior to v0.38 have been removed from the CLI `migrate` command. The oldest supported migration is v0.39->v0.42.
-
+* [\#9371](https://github.com/cosmos/cosmos-sdk/pull/9371) Non-zero default fees/Server will error if there's an empty value for min-gas-price in app.toml
```

Comment:
This should just be changed because of #9621 right?

Reply:
That was part of the RC-0 release, we had it there and it was deleted in the previous commit. So instead I put it back and added:
https://github.com/cosmos/cosmos-sdk/pull/9700/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR43

Reply:
Okay, I assume we'll clean this up for the final release notes right?

Reply:
yes - this is what I was asking in Discord.